crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -59,6 +59,7 @@ impl TransactionBlock {
             return Ok(None);
         }
         let gcs = self.effects.as_ref().unwrap().gas_effects.gcs;
+        // TODO: implement DB counterpart without using Sui SDK client
         let data_provider = ctx.data_provider();
         let system_state = data_provider.get_latest_sui_system_state().await?;
         let protocol_configs = data_provider.fetch_protocol_config(None).await?;
```

Comment:
@wlmyng I wouldn't consider transaction block done while these are here.
Is the DB impl feasible yet?

Reply:
@stefan-mysten is working on this (transactionBlock.expiration) field, should be doable as expiration is actually a field returned from StoredTransaction

Reply:
Yes, a PR should come in shortly once I figure some really weird errors.

Reply:
#14147